src/styles/blocks/button.scss

```diff
@@ -0,0 +1,21 @@
+.card__btn_link {
```

Comment:
Move this inside `.card` as well
Create files only for BEM blocks

Reply:
done.

thank you so much for your comments, it makes me better.

Please explain in more detail, why the button cannot be a block, because theoretically we can reuse it in the project in the future??